### PR TITLE
Implement kited autostart

### LINKED ISSUE #180: Autostart kited at startup
Autostart kited at startup of the plugin.

- if necessary add a new setting to disable the automatic startup, default to false, i.e. to an automatic start of kited
- if setting "start kited at startup" is enabled:
    - detect if kited is running at startup
         - if not running: detect if kited is installed
            - if installed: **launch kited**

### PR DESCRIPTION
**Pull request by abe33**
Closes #180

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -98,6 +98,11 @@
                     "default": false,
                     "description": "Displays JSON data used by a view"
                 },
+                "kite.startKitedAtStartup": {
+                    "type": "boolean",
+                    "default": false,
```
[the remark this answers is not in the document]
Comment (abe33) [reply package.json]: Hmm, then the wording in the issue is really, really confusing: 
> if necessary add a new setting to disable the automatic startup, **default to false**, i.e. to an automatic start of kited

**Comment by abe33** [comment]
> Kite seems to be terminated when a VSCode which started Kite is terminated. Is it possible to keep kited running in such a case?

I don't reproduce on OSX, could it be a windows specific issue? If so here's the snippet we use to start Kite on windowd: https://github.com/kiteco/kite-connect-js/blob/master/lib/support/windows.js#L218-L230 do you know a better way to start it?

**Comment by jansorg** [comment]
@abe33 I'm not seeing this on macOS, so it seems to be winodws-only.
Is `unref` https://nodejs.org/api/child_process.html#child_process_subprocess_unref useful here?

**Comment by abe33** [comment]
@jansorg so `unref` isn't working neither. I guess we'll go with the current version that terminates kited on close.

**Comment by jansorg** [comment]
I'd prefer not to kill Kite, but as there's no other solution with VSCode's limitations feel free to merge.

**Pull request merged**